app/js/main.js

```diff
@@ -18,8 +18,24 @@ import {DemoPaywallController} from './demo-controller';
 import {MeteringDemo} from './metering';
 import {log} from './log';
 
+/* eslint-disable */
+const REGWALL = GaaMeteringRegwall;
+/* eslint-enable */
+
 log('started');
 
+function hasPublisherSubscription() {
```

Comment:
These new functions seem to make the code less readable, for a few reasons. First, they're only used in one place each. Second, that place is pretty far from where the functions are defined. Third, the functions don't have comments